potpie/observability/observability/__init__.py

```
@@ -44,15 +44,7 @@
 from .sink import Sink
 
-__all__ = [
-    "get_logger",
-    "configure",
-    "log_context",
-    "bind_context",
-    "reset_context",
-    "ObservabilityConfig",
-    "Sink",
-]
+__all__ = ["get_logger", "configure", "log_context", "ObservabilityConfig", "Sink"]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)